client-src/elements/chromedash-feature-page.ts

```diff
@@ -70,41 +81,34 @@ export class ChromedashFeaturePage extends LitElement {
     ];
   }
 
-  static get properties() {
-    return {
-      user: {attribute: false},
-      paired_user: {attribute: false},
-      featureId: {type: Number},
-      feature: {type: Object},
-      featureLinks: {type: Array},
-      gates: {type: Array},
-      comments: {type: Array},
-      process: {type: Object},
-      progress: {type: Object},
-      contextLink: {type: String},
-      appTitle: {type: String},
-      starred: {type: Boolean},
-      loading: {attribute: false},
-      selectedGateId: {type: Number},
-    };
-  }
-
-  constructor() {
-    super();
-    this.user = null;
-    this.featureId = 0;
-    this.feature = {};
-    this.featureLinks = [];
-    this.gates = [];
-    this.comments = {};
-    this.process = {};
-    this.progress = {};
-    this.contextLink = '';
-    this.appTitle = '';
-    this.starred = false;
-    this.loading = true;
-    this.selectedGateId = 0;
-  }
+  @property({type: Object, attribute: false})
```

Comment:
Per [@jyasskin's comment](https://github.com/GoogleChrome/chromium-dashboard/pull/4089#discussion_r1672824243), these should likely be changed to have either 1 of `type` or `attribute`.